### ISSUE TITLE
Admin static media has disappeared from S3

### ISSUE THREAD

**Issue by ejucovy** [350dotorg/localpower#62]
At some point in the course of my deployments to the staging site the static media in the S3 `rah_django_admin_media` bucket has disappeared.

Grepping the code I don't see anything that actually pushes these assets to that (or any) bucket.  Were we just relying on something that was previously set up for repowerathome.com and has perhaps subsequently been deleted?

**Comment by ejucovy** [comment]
> Were we just relying on something that was previously set up for repowerathome.com and has perhaps subsequently been deleted?

(I suspect this to be the case, particularly since S3 gives an error message saying "the specified bucket does not exist".)

**Comment by ejucovy** [comment]
Actually it's even a little weirder than that; the nginx conf files in `fabfile` don't reference `rah_django_admin_media` at all.  But, the conf files in `fabfile_sample` _do_.  For some reason we seem to be getting the nginx `rah_base` conf file from `fabfile_sample` instead of from `fabfile`.

**Comment by jonlesser** [comment]
A long long time ago I decided to serve django's admin media out of a separate S3 bucket from the main static files buckets. You are right that the admin files had been serving out of a bucket associated with RAH's aws account that I closed down last week. 

I believe we're pulling rah_base from the right place. I had made some edits to the fabfile between the time that I used it to launch a dev server and the time I checked it all in to github. This is what I did:

1) Launched dev server
2) Remembered admin media stuff
3) Created a bucket with the files and edited rah_base in fabfile
4) checked in fabfile
5) Everything worked, but I didn't realize admin media was actually being served from RAH account. 

Anywho, I edited the rah_base file on the dev server so it's pointing to the right bucket.
